apps/nextjs-website/react-components/components/Hero/Hero.helpers.tsx

```diff
@@ -49,23 +49,27 @@ export const HeroTextContent = ({
       alignItems={size === 'small' ? 'center' : 'inherit'}
       sx={{
         minHeight: 'inherit',
-        padding: size === 'small' ? '60px 0' : '0',
-        maxWidth: size === 'small' ? '60%' : '100%',
+        paddingY: size === 'small' ? '60px' : '0',
+        maxWidth: { xs: '100%', md: size === 'small' ? '60%' : '100%' },
         marginTop: size === 'small' ? '0' : { xs: '3em', lg: '0' },
         ...(size === 'small' && { margin: '0 auto' }),
       }}
       component='section'
       spacing={2}
     >
       <Stack spacing={2} mb={size === 'small' ? 0 : { xs: 6, md: 4 }}>
-        <Title
-          variant='h1'
+        <Typography
           component={titleTag}
-          textColor={textColor}
-          title={title}
+          variant='h1'
           textAlign={size === 'small' ? 'center' : 'left'}
-          marginBottom={size === 'small' ? 0 : 2}
-        />
+          sx={{
+            fontSize: { xs: '36px', md: '50px' },
+            color: textColor,
+            marginBottom: size === 'small' ? 0 : 2,
+          }}
```

Comment:
```suggestion
          fontSize={{ xs: '36px', md: '50px' }}
          textColor={textColor}
          marginBottom={size === 'small' ? 0 : 2}
```
Try avoiding the use of sx wherever possible